### DISCUSSION TITLE
Hajk 3.7 Release Candidate 1

### DISCUSSION THREAD

**Discussion by Hallbergs** [hajkmap/Hajk#857] [General]
_[114 commits](https://github.com/hajkmap/Hajk/compare/v3.6.0...v3.7.0-rc.1) since 3.6, here's Hajk 3.7 RC1! This release was brought to you by @Hallbergs  and @jacobwod . Please see the list of commits for all contributors._

# Changelog

## Major new features
- Client: New plugin (Timeslider), allows for users to see how data changes over time (https://github.com/hajkmap/Hajk/wiki/Tidslinje)
- Client: New plugin (VTsearch), allows for timetables searches (https://github.com/hajkmap/Hajk/wiki/Västtrafik)
- Backend: Add a generic proxy to the backend (#824). This replaces the .NET proxy as it allows Hajk backend to proxy any service (exposed on any port), via a path on Hajk's API. Example usage includes solving CORS problems or proxying requests to underlying services running on different port (e.g. QGIS Server). 
- Client: **WMS layers** can be used as **search sources**, only minimal configuration is required to enabled search functionality for WMS (where a corresponding WFS service exists), (22d648e)
- Client: Add support for WFS-T, including editing, from QGIS Server (#825)

## New features
- Search results labels in map can be turned off on a per-user basis, setting is stored locally in user's browser (#773)
- WFS-T layers can be restricted to certain AD groups (#762)
- **Min/max zoom** restrictions on layers are now visualized in client so that user knows that the layer is on, but outside current zoom (#788)
- **New print** solution for the **DocumentHandler** plugin (#796)
- Add support for **`dynamic` Window height** in Admin (#787)
- Automatically **turn visibility on** for WMS search layers that got clicked (#816)
- **More filters** in the feature-info dialog (#820)
- Add **search phrase to Anchor link** (#830)

## Bug fixes
- FeatureInfo: upgrade to ReactMarkdown 6.0 (#761)
- Better infoclick (FeatureInfo) behavior (#771)
- Unicode characters allowed in infoclick defintion placeholders (64bf6fa) 
- Breadcrumbs respect the chosen theme (light/dark) (#802)
- Host Google Fonts locally (#809)
- Minor bugfixes: #807, #812, #817, #822, #826, #831, #832, #836, #838, #839, #851, #852. 

---

# Downloads
There are multiple ways to get the latest Hajk up and running. For the first time ever, we even provide a Docker container!

## Pre-compiled releases
This time we have **three different** version of the pre-compiled solution (scroll down for download links):
1. [Hajk 3.7.0 RC1 NodeJS edition](https://github.com/hajkmap/Hajk/releases/download/v3.7.0-rc.1/hajk-v3.7.0-rc.1-nodejs.zip)
1. [Hajk 3.7.0 RC1 Simple edition](https://github.com/hajkmap/Hajk/releases/download/v3.7.0-rc.1/hajk-v3.7.0-rc.1-simple.zip)
1. [Hajk 3.7.0 RC1 .NET edition](https://github.com/hajkmap/Hajk/releases/download/v3.7.0-rc.1/hajk-v3.7.0-rc.1-dotnet.zip)

## Docker container
The container is available on Docker Hub: https://hub.docker.com/r/hallbergs/hajk

# Installation
Refer to our recently updated [installation guide](https://github.com/hajkmap/Hajk/wiki/Installation-guide-%28for-pre-packaged-releases%29).

<hr /><em>This discussion was created from the release <a href='https://github.com/hajkmap/Hajk/releases/tag/v3.7.0-rc.1'>Hajk 3.7 Release Candidate 1</a>.</em>

**Comment by ingvar-uddevalla** [comment]
UDDEVALLA: When creating a new map, it's only the projection EPSG:3006 (Sweref 99 TM) which is inserted in the <newmap>.json file. The remaining projections, for example 3007 has to be manually inserted afterwards. This seems to be the case in earlier releases too. I don't know if itis a bug or a lack of definition in other config-files. However all projections are defined in layers.json.

**Reply by Hallbergs** [reply]
Nice catch! We've accidentally shipped the releases with an incorrect map-template. Will be fixed in RC2!

**Comment by emmafrygne-vbg** [comment]
VARBERG: 
- We are missing the Timeslider plugin in Admin. 
- I can´t write text (only numbers) in window height in Admin so I can´t create dynamic windows. 

We are using the .Net edition and I have not copied the new config in admin.

**Reply by Hallbergs** [reply]
Thanks for the info! Please test RC2, the problems should be fixed :)

**Reply by emmafrygne-vbg** [reply]
I have tested RC2 and the problems are fixed :) Thank you!

**Comment by Hallbergs** [comment]
The template-problem and the problem with missing TimeSlider in admin is fixed in RC2, see: https://github.com/hajkmap/Hajk/releases/tag/v3.7.0-rc.2

Also, now you can type text in the height field, allowing for dynamic windows!
